Sources/PodToBUILD/MultiPlatform.swift

```diff
@@ -206,7 +206,7 @@ public struct AttrSet<T: AttrSetConstraint>: Monoid, SkylarkConvertible, EmptyAw
     }
 
     public func partition(predicate: @escaping (T) -> Bool) -> (AttrSet<T>, AttrSet<T>) {
-        return (self.filter(predicate), self.filter((!) • predicate))
+        return (self.filter(predicate), self.filter { x in !predicate(x) })
```

Comment:
When I was working on a feature this morning, I couldn't figure out that `(!) • predicate)` meant
> Invert the predicate

The original function violated the [Rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) via an operator represented by a character I didn't see on the Apple keyboard. Removing this is a part of #37 to clean things up a bit.